src/libs/ui/components/list/list.tsx

```diff
@@ -77,12 +79,14 @@ export function List<ListRow extends ListRowBase>({
   renderFooter,
   headerLabel,
   headerAction,
-  marginLeftForItemSeparatorLine
+  marginLeftForItemSeparatorLine,
+  headerLabelSpaceSize = SpacingSize.ExtraLarge,
```

Comment:
This should be renamed to top, as there is functionally no difference from existing `top` prop on the existing `VerticalSpaceContainer`, consistency should be the most valuable priority.

Reply:
cool make sense 👍